packages/fxa-payments-server/src/components/PaymentConfirmation/index.tsx

```diff
@@ -160,7 +160,7 @@ export const PaymentConfirmation = ({
 
         <div className="footer" data-testid="footer">
           <PaymentLegalBlurb provider={payment_provider} />
-          <TermsAndPrivacy plan={selectedPlan} showFXALinks={true} />
+          <TermsAndPrivacy plan={selectedPlan} />
```

Comment:
We want to pass `showFXALinks={!accountExists}` and `contentServerURL` just like for `Checkout`. The FxA links will never show on `PaymentConfirmation` otherwise.